include/seqan/misc/bit_twiddling.h

```diff
@@ -514,7 +514,7 @@ _bitScanForward(TWord word, WordSize_<NUM_BITS>)
     return _bitScanForwardGeneric(word, WordSize_<NUM_BITS>());
 }
 
-#if defined(PLATFORM_GCC)
+#if defined(COMPILER_GCC) || defined(COMPILER_INTEL) || defined(COMPILER_CLANG)
```

Comment:
The windows implementation is further below. I don't really understand what you mean. 

For example on linux:

```
 _bitScanReverse(TWord word, WordSize_<64>)
 {
     return 63 - __builtin_clzll(static_cast<unsigned long long>(word));
 }
```

For example on windows (see also [here](http://stackoverflow.com/a/20468180)):
```
 template <typename TWord>
 inline TWord
 _bitScanReverse(TWord word, WordSize_<64>)
 {
     unsigned long index;
     _BitScanReverse64(&index, static_cast<uint64_t>(word));
     return index;
 }
```

See also: https://msdn.microsoft.com/en-us/library/bb384809.aspx

> On Intel processors that do not support the lzcnt instruction, the instruction byte encoding is executed as bsr (bit scan reverse). If code portability is a concern, consider use of the _BitScanReverse intrinsic instead. For more information, see _BitScanReverse, _BitScanReverse64.